pyramid_hypernova/context_manager.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from contextlib import contextmanager
+
+from pyramid_hypernova.rendering import RenderToken
+from pyramid_hypernova.tweens import configure_hypernova_batch
+
+
+@contextmanager
+def hypernova_batch(request, registry):
+    """A context manager that performs hypernova token replacement in a batch.
+    Write the content you wish to modify in body['content'] where body is the
```

Comment:
If you don't like yielding the mutable `body`, you _could_ wrap it will a closure. Maybe this interface is a little nicer? :\

```
@contextmanager
def hypernova_batch(...):
    body = { 'content': None }

    def set_content(content):
        body['content'] = content

    yield set_content



with hypernova_batch(...) as set_content:
    resp = do_the_rendering_stuff()
    set_content(resp.content)
```

Reply:
Hmm, but wit this, the consuming code won't be able to retrieve updated `body` after the context manager exits. I'll keep it as is.